packages/packages/commonwell-sdk/src/client/commonwell.ts

```diff
@@ -99,6 +100,25 @@ export class CommonWell {
     return personSearchRespSchema.parse(resp.data);
   }
 
+  /**
+ * Searches for a person based on patient demo.
+ * See: https://specification.commonwellalliance.org/services/patient-identity-and-linking/protocol-operations#8713-find-persons-matching-patient-demographics
+ *
+ * @param meta    Metadata about the request.
+ * @param id      The patient to be updated.
+ * @returns
```

Comment:
General comment - most of these new function headers are lacking some indentation